src/skin/legacyskinparser.cpp

```diff
@@ -639,6 +636,23 @@ QWidget* LegacySkinParser::parseWidgetGroup(const QDomElement& node) {
         }
     }
     m_pParent = pOldParent;
+}
+
+QWidget* LegacySkinParser::parseWidgetGroup(const QDomElement& node) {
+    WWidgetGroup* pGroup = new WWidgetGroup(m_pParent);
```

Comment:
We must not mix smart pointers with the Qt object tree, which manages the widget lifetime just right. 

This widget is added to the object tree here: 
https://github.com/mixxxdj/mixxx/pull/1068/files/9b7ac9a01c34812922b73f48ec328eba820b0433#diff-c2c84548a4801a7ca8649f1095e0bf36L601

Some time ago we have tested the object tree performance versus smart pointer solutions. 
The object tree has won.